executor/runtime/docker/seccomp/Makefile

```diff
@@ -3,5 +3,7 @@
 all: seccomp.go
 
 seccomp.go: default.json fuse-container.json
-	gobin -run -m github.com/jteeuwen/go-bindata/go-bindata -pkg seccomp -o seccomp.go default.json fuse-container.json
+	python3 -m json.tool default.json
```

Comment:
Is this just for debugging purposes?

Reply:
This is for consistent formatting purposes.